src/app/lists/combo/simple-combo-usage/simple-combo-usage.component.ts

```diff
@@ -0,0 +1,17 @@
+import * as core from '@angular/core';
```

Comment:
The way I see it in every component file of the simple combo we're importing * from angular/core.